guide/case-studies/wallet-scrutiny.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: guide
+title: Wallet Scrutiny
+description: This was a collaboration between Wallet Scrutiny and the Bitcoin Design Community to help with the redesign of the website.
+nav_order: 4
+parent: Case studies
+permalink: /guide/case-studies/wallet-scrutiny/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-preview.jpg
+---
+
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/wallet-scrutiny/banner-header-wallet-scrutiny.png"
+   retina = "assets/images/guide/case-studies/wallet-scrutiny/banner-header-wallet-scrutiny@2x.png"
+   alt-text = "A banner image with two screenshots of the redesigned homepage including the homepage and the individual wallet review page."
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Wallet Scrutiny
+
+## How it started
+
+In November 2022 the Wallet Scrutiny project was introduced to the Bitcoin design Community. They are a small team producing an extensive amount of work and are on a mission to improve their website and increase their reach in the ecosystem. The Bitcoin Design community kicked off a redesign process with the project to help them reach their goals. The overarching goal was to redesign the website with the goal of improving the user experience.
+
+## What does Wallet Scrutiny do?
+
+Wallet Scrutiny is a website that helps everyday bitcoin users verify whether or not their wallet is truly open-sourced and secure. They aim to improve the security of all bitcoin wallets by open sourcing the evaluation of their source code. They do this by focusing on the reproducibility of wallets, which ensures that the wallet a regular user uses actually matches with the code published online.
+
+They:
+- Produce security verdicts on 3000+ mobile and hardware wallets
+- Write in-depth documentation on the analysis of each wallet
+- Maintain a high-quality security review methodology
+- Provide navigation of wallets and verdicts via search
+
+
+## The process
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/wallet-scrutiny/process-wallet-scrutiny.png"
+   retina = "assets/images/guide/case-studies/wallet-scrutiny/process-header-wallet-scrutiny@2x.png"
+   mobile = "assets/images/guide/case-studies/wallet-scrutiny/process-header-wallet-scrutiny-mobile.png"
+   alt-text = "An image of the seven steps that were taken during the re-design process. Those steps were discovery, ux plan, user groups, baseline testing, re-branding, re-design, re-testing"
```

Comment:
UX should be capitalized, I think. Not sure about the "re-" thingies. Doesn't look quite right and I'd probably go for "redesign", "rebranding", etc. but I'm not a native speaker.

Reply:
Good spot, edited these to be one word also in the headings.